.eslintrc.yml

```
@@ -0,0 +1,33 @@
+root: true
+
+env:
+  browser: true
+  node: true
+  es2021: true
+
+parserOptions:
+  ecmaVersion: 2022
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+extends:
+  - eslint:recommended
+  - plugin:import/recommended
+  - plugin:react/recommended
+  - plugin:react-hooks/recommended
+
+rules:
+  import/first: error
+  import/newline-after-import: error
+  import/no-duplicates: error
+  no-console: error
+  react/prop-types: off
+
+
+settings:
+  import/resolver:
```

This should probably be a string
```suggestion
  import/resolver: "build-scripts/eslint-resolver.cjs"
```